providers/apis/geckoterminal/api_handler_test.go

```
@@ -1,278 +1,285 @@
+			handler := &APIHandler{
+				api: config.APIConfig{
+					Name:    Name,
+					Enabled: true,
+				},
+			}
 
-			// Update the cache since it is assumed that createURL is executed before ParseResponse.
-			_, err = h.CreateURL(tc.cps)
-			require.NoError(t, err)
+			// Create a test server
+			server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

golangci-lint (1.64.8)

[warning] unused-parameter: parameter 'r' seems to be unused, consider removing or renaming it as _

(revive)